peering/forms.py

```diff
@@ -787,6 +789,7 @@ class RoutingPolicyForm(BootstrapMixin, forms.ModelForm):
     slug = SlugField(max_length=255)
     type = forms.ChoiceField(choices=RoutingPolicyType.choices, widget=StaticSelect)
     address_family = forms.ChoiceField(choices=IPFamily.choices, widget=StaticSelect)
+    config_context = JSONField(required=False, label="Context", widget=SmallTextarea)
```

Comment:
Maybe using `Config context` as label instead of just `Context` for better understanding.